### PR TITLE
Add group features and update certs

### PR DESCRIPTION
**Pull request by demoore**
PEM file is now updated to work with future sendwithus SSL certs

### CODE REVIEW

File: lib/API.php [hunk 1]
```diff
@@ -22,7 +22,7 @@ class API {
     private $API_VERSION = '1';
     private $API_HEADER_KEY = 'X-SWU-API-KEY';
     private $API_HEADER_CLIENT = 'X-SWU-API-CLIENT';
-    private $API_CLIENT_VERSION = "2.3.1";
+    private $API_CLIENT_VERSION = "2.4.1";
```
Comment (brandonb927) [inline lib/API.php]: I think this should be `2.4.0`, no?

File: lib/API.php [hunk 2]
```diff
@@ -557,7 +596,6 @@ private function api_request($endpoint, $request = "POST", $payload = null, $par
         }
         else {
             $httpheaders = array(
-                'Content-Type: application/json',
```
Comment (brandonb927) [inline lib/API.php]: Pretty sure this header needs to exist here?

File: lib/API.php (hunk 1 above)
Comment (demoore) [inline lib/API.php]: Good point!

File: lib/API.php (hunk 2 above)
Comment (demoore) [inline lib/API.php]: You don't need to set the content if there is no request body (`$payload`)

Comment (brandonb927) [inline lib/API.php]: ah, ok. Carry on.

**Comment by brandonb927** [comment]
Once the tests on Travis pass, :+1:

**Pull request merged**